service/matching/matching_engine.go

```diff
@@ -904,10 +906,13 @@ func (e *matchingEngineImpl) UpdateWorkerVersioningRules(
 				req.GetDeleteCompatibleRedirectRule(),
 			)
 		case *workflowservice.UpdateWorkerVersioningRulesRequest_CommitBuildId_:
+			// hasRecentPoller := tqMgr.GetVersionedPhysicalTaskQueueManager(req.GetCommitBuildId().GetTargetBuildId()).HasPollerAfter(time.Now().Add(-versioningPollerSeenWindow))
```

Comment:
I was thinking that after moving out the versioned pollers into their own physicalTaskQueueManagers, we could check the recent poller like this